ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -274,7 +275,7 @@ public TransactionProcessingResult processTransaction(
         return TransactionProcessingResult.invalid(validationResult);
       }
 
-      final MutableAccount senderMutableAccount = sender.getMutable();
+      final var senderMutableAccount = sender.getMutable();
```

Comment:
I'm pretty happy using `var` everywhere but not sure if everyone else is?

Reply:
Worth discussing next contributor call